roles/pulp_common/README.md

```diff
@@ -132,7 +132,7 @@ If it is set to "packages", the following variables are used, or behave *differe
 * `pulp_install_dir`: Location of the filesystem prefix where package installed python programs
   (gunicorn & rq) are looked for on the filesystem.  Defaults to "/usr" (such as for "/usr/bin/gunicorn").
 * `pulp_django_admin_paths`: A list of possible filepaths to the `django-admin` command.
-  The first one found will be used for pulp_installer's internal usage. See `defaults/main.yml` for default values.
+  The last one found will be used for pulp_installer's internal usage. See `defaults/main.yml` for default values.
```

Comment:
Are you sure the last one found is used? The loop is supposed to run until `pulp_django_admin_path` is defined. The setting of `pulp_django_admin_path` is supposed to break the loop (or rather, cause future iterations to be skipped.)

The output lists "OK" for all iterations under release-static for the set_fact task, which makes me think it isn't working.

Reply:
Needless to say, that i prefer that very much, as every occurrence of `set_fact` makes the `--start-at-task` parameter unusable.